exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -162,46 +179,42 @@ impl JavaServiceRuntime {
 
     /// Adds required EJB-related arguments to JVM configuration
     fn add_required_arguments(
-        args_builder: InitArgsBuilder,
-        ejb_config: EjbConfig,
+        mut args_builder: InitArgsBuilder,
+        runtime_config: &RuntimeConfig,
     ) -> InitArgsBuilder {
+        // We do not use system library path in tests, because an absolute path to the native
```

Comment:
It might be provided to the system linker, but JVM still requires `java.library.path`, doesn't it? Also, how tests are different from the the application? Both will have the library on the runpath.

Reply:
Same here.